extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-rtc.js

```diff
@@ -147,4 +147,28 @@ describes.realWin('DoubleClick Fast Fetch RTC', {amp: true}, env => {
     });
 
   });
+
+  describe('rewriteRtcKeys', () => {
```

Comment:
I would have expected a need to change other tests in terms of their resulting json targeting, no?